Fix:
warning: incompatible pointer types passing 'unsigned int [2]' to parameter of type 'const struct utimbuf *' [-Wincompatible-pointer-types]
--- unzoo.c.orig	2002-05-06 10:34:00.000000000 -0500
+++ unzoo.c	2022-01-05 07:30:56.000000000 -0600
@@ -618,8 +618,8 @@
 **  members with the correct time as stored in the archive.
 */
 #ifdef  SYS_IS_UNIX
-unsigned int   Secs [2];
-#define SETF_TIME(patl,secs)    (Secs[0]=Secs[1]=(secs),!utime((patl),Secs))
+struct utimbuf Secs;
+#define SETF_TIME(patl,secs)    (Secs.actime=Secs.modtime=(secs),!utime((patl),&Secs))
 #endif
 #ifdef  SYS_IS_DOS_DJGPP
 unsigned long   Secs [2];
